pkg/sdk/dynamic_table_integration_test.go

```diff
@@ -0,0 +1,96 @@
+package sdk
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestInt_DynamicTableCreate(t *testing.T) {
```

Comment:
Please add tests for alter and drop

Reply:
added for alter, i am not sure if we need one for drop. i have renamed the create to "TestInt_DynamicTableCreateAndDrop" to make it more clear